doc/libpmem.3.md

```diff
@@ -164,6 +164,11 @@ appropriate for flushing changes to persistence. Calling
 **pmem_is_pmem**() each time changes are flushed to persistence will
 not perform well.
 
+>NOTE:
```

Comment:
How about adding an additional sentence to this note to give people a little hope.  "In the future, ss file systems become available that support flushing with **pmem_persist**(), **pmem_is_pmem**() will return true as appropriate."

Also "directly from DAX Device" should be "directly from Device DAX" since that's how the ndctl man page refers to it.